Instrukcije_d001/src/app/terminal.c

```diff
@@ -0,0 +1,131 @@
+//
+// Created by Edin on 15. 03. 2022.
+//
+
+#include "terminal.h"
+
+int stevec = 0; // counter za to kdo bo na vrsti (X ali 0)
+int igreJeKonec = 0; // flag za konec igre
+
+void terminal_init() {
+    initscr();
+    curs_set(0);
+
+    noecho();
+    refresh();
+}
+
+// Zakaj memoTabela mora bit const?
+void jeKdoZmagal(int tabela[ST_ZMAGOVALNIH_KOMBINACIJ][3], int memoTabela[WORLD_ST_ZETONOV]) {
+    for (int i = 0; i < ST_ZMAGOVALNIH_KOMBINACIJ; ++i) {
+        if ((memoTabela[tabela[i][0]] > 0 && memoTabela[tabela[i][1]] > 0 && memoTabela[tabela[i][2]] > 0)
+            && (memoTabela[tabela[i][0]] == memoTabela[tabela[i][1]])
+            && (memoTabela[tabela[i][0]] == memoTabela[tabela[i][2]])) {
+            igreJeKonec = 1;
+            mvprintw(8, 0, "Game over!\nPlayer %c has won the game!\n"
```

Comment:
Če ta funkcija spada v domain layer potem bo potrebno `terminal` specific funkcije nekako prestaviti ven iz te funkcije, da bo lahko to funkcijo uporabljal tudi Android ki ne pozna mvprintw, to pozna samo terminal.